hw/ip/aes/data/aes.hjson

```diff
@@ -151,6 +151,8 @@
       AES unit is non-idle, writes to this register are ignored.
       This register is shadowed, meaning two subsequent write operations are required to change its content.
       If the two write operations try to set a different value, a ctrl_err alert is triggered.
+      Any write operation to this register will clear the status tracking required for automatic mode (See MANUAL_OPERATION field).
+      After a write to the Control Register, software needs to rewrite Initial Key, IV and Input Data Registers.
```

Comment:
a different suggestion that maybe a bit easier to understand. (rewrite sounds like putting in the previous values)

> A write to the control register is considered the start of a new operation - hence new key, Iv and data need to be provided.

Reply:
That's a very good suggestion, I will adapt it.